sky/tools/sky_snapshot/main.cc

```diff
@@ -63,5 +79,13 @@ int main(int argc, const char* argv[]) {
   CHECK(command_line.HasSwitch(switches::kSnapshot)) << "Need --snapshot";
   WriteSnapshot(command_line.GetSwitchValuePath(switches::kSnapshot));
 
+  if (command_line.HasSwitch(switches::kDepfile)) {
+    CHECK(command_line.HasSwitch(switches::kBuildOutput)) <<
+        "Need --build-output with --depfile";
+    WriteDependencies(command_line.GetSwitchValuePath(switches::kDepfile),
+                      command_line.GetSwitchValueASCII(switches::kBuildOutput),
```

Comment:
Not a strong opinion, but doing that could lead to a wrong use because depfile is a very ninja specific format.

Changing this to what you suggested unless you feel otherwise.